OpenRA.Mods.Common/Traits/HitShape.cs

```diff
@@ -35,6 +35,7 @@ public class HitShapeInfo : ConditionalTraitInfo, Requires<BodyOrientationInfo>
 		public readonly BitSet<ArmorType> ArmorTypes = default(BitSet<ArmorType>);
 
 		[FieldLoader.LoadUsing("LoadShape")]
+		[Desc("Comes with support for `Circle`, `Polygon` and `Rectangle`. Defaults to `Circle` when left empty.")]
```

Comment:
Mods can add their own, so this list isn't necessary complete.

Reply:
I know.